templates/absences/edit.html.twig

```diff
@@ -18,15 +18,17 @@
   <h3>Modification de l'absence</h3>
   {% if not access %}
       Vous n'êtes pas autorisé(e) à modifier cette absence.<br/><br/>
-      <a href='{{ asset("index.php?page=absences/voir.php") }}'>Retour à la liste des absences</a><br/><br/>
+      <a href='{{ asset("/absence") }}'>Retour à la liste des absences</a><br/><br/>
   {% else %}
-    <form name='form' id='form' method='get' action='/index.php' onsubmit='return verif_absences(\"debut=date1;fin=date2;motif\");'>
+    <form name='form' id='form' method='post' action='{{ asset("/absence") }}' onsubmit='return verif_absences(\"debut=date1;fin=date2;motif\");'>
       <input type='hidden' name='CSRFToken' value='{{ CSRFToken }}' />
       <input type='hidden' name='page' value='absences/modif2.php' />
       <input type='hidden' name='perso_id' value='{{ absence.perso_id }}' />
       <input type='hidden' id='login_id' value='{{ loggedin_id }}' />
       <input type='hidden' name='groupe' id='groupe' value='{{ absence.groupe }}' />
+      <input type='hidden' name='absence_id' id='absence_id' value='{{ id }}' />
```

Comment:
There is already an input with name id at the end of the form. This line is useless